### PR TITLE
DeploymentConfig with the optional env_variables_prefix parameter, re…

### PR DESCRIPTION
**Pull request by beczkowb**
…moving that parameter from the Config class

### CODE REVIEW

**Comment by iwko** [comment]
Why you've created separate class for DeploymentConfig?

**Comment by beczkowb** [comment]
@iwko This is a solution for #99 . Environment variable prefix works only for the `bf deploy` command (because we need the prefix for bamboo). In any other case we assume that the prefix == `bf_`. So it's misleading for a user to have env prefix parameter in the Config class.

**Review by iwko — APPROVED** [review]

File: bigflow/configuration.py [hunk 1]
```diff
@@ -111,3 +114,16 @@ def _resolve_placeholders(self, value, variables: dict):
...
             return value
 
+
+class DeploymentConfig(Config):
+    def __init__(self,
+                 name: str,
+                 properties: dict,
+                 is_master: bool = True,
+                 is_default: bool = True,
+                 environment_variables_prefix: str = DEFAULT_CONFIG_ENV_VAR_PREFIX):
```
Comment (anjensan) [inline bigflow/configuration.py:124]: nit: consts are not actual consts in PY, so someone might change variable 'DEFAULT_CONFIG_ENV_VAR_PREFIX' and will not be reflected on this class/parameter. Maybe `None` OR drop the 'const variable'.

**Review by anjensan — APPROVED** [review]
In general approve.

*Maybe* just add this as a "hidden feature" to base 'Config' class
In fact this feature is already here, as you can write
```python
c = bigflow.Config("name", {})
c.environment_variables_prefix = "foo_"
```

It is not documented neither for Config not for DeployConfig...
So it is hidden from user in any case.

File: bigflow/configuration.py (hunk 1 above)
Comment (beczkowb) [reply bigflow/configuration.py:124]: done

**Comment by beczkowb** [comment]
@anjensan I will update the docs later, now we want to finish the code so we can publish RC1.

**Pull request merged**